selftests/functional/test_basic.py

```diff
@@ -1141,7 +1141,7 @@ class PluginsXunitTest(AbsPluginsTest, unittest.TestCase):
     def setUp(self):
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         self.junit = os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                     os.path.pardir, ".data", 'junit-4.xsd'))
+                                                  os.path.pardir, ".data", 'junit-4.xsd'))
```

Comment:
This also exceeds the 80 chars. Would you please assign to a variable, or break the line?